tools/split.py

```diff
@@ -134,6 +157,11 @@ def main():
             talk_path = os.path.join(subdir_path, talk_name)
             thumbnail_path = os.path.join(subdir_path, image_name)
 
+            youtube_desc = make_video_description(talk, talk["description"])
```

Comment:
I think these should be done further up all at once prior to downloading / cutting any video. With potentially long-processing scripts, as much validation should be done upfront as possible, instead of potentially having the user come back from dinner to see that it threw an exception after processing one video.

I'm not sure if it'll be used, but this also makes it cleaner to offer a cmdline arg to not do this validation, say if the user really doesn't care about fixing descriptions right now. Letting the validation in upload.py stay is fine.